src/common/offline_transformations/src/compress_quantize_weigths.cpp

```diff
@@ -70,7 +70,9 @@ ngraph::pass::CompressQuantizeWeights::CompressQuantizeWeights() {
         auto fq_users = fq->get_users();
         if (fq_users.size() == 1 && has_dequantization_subgraph(fq_users[0])) {
             auto& first_convert = fq_users[0];
+            OPENVINO_SUPPRESS_DEPRECATED_START
             if (auto new_weights = ov::get_constant_from_source(first_convert)) {
+                OPENVINO_SUPPRESS_DEPRECATED_END
```

Comment:
Looks like will be easier just to wrap all if body inside...
```
OPENVINO_SUPPRESS_DEPRECATED_START
if (...) {
} else {
}
OPENVINO_SUPPRESS_DEPRECATED_END
```
In this case we have the same indentation.